ui/imports/shared/controls/ExpandableTag.qml

```diff
@@ -0,0 +1,112 @@
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+import QtQuick.Layouts 1.15
+
+import StatusQ.Core 0.1
+import StatusQ.Components 0.1
+import StatusQ.Core.Theme 0.1
+
+import utils 1.0
+
+Control {
+    id: root
+    width: 120
+    height: 50
+    anchors.left: parent.left
+    anchors.leftMargin: 12
+    anchors.bottom: parent.bottom
+    anchors.bottomMargin: 12
+
+    property string tagHeaderText
+    property string tagName
+    property string tagImage
+    property bool isIcon: false
+    property color backgroundColor: Theme.palette.baseColor4
+
+    property bool expanded: root.hovered
+
+    signal tagClicked()
+
+    contentItem: Item {
+
+        StatusBaseText {
+            id: textLabel
+            width:  parent.width
+            visible: root.expanded && !!root.tagHeaderText
+            opacity: visible ? 1 : 0
+            color: Theme.palette.indirectColor1
+            Behavior on opacity { NumberAnimation {} }
+            verticalAlignment: Text.AlignVCenter
+            text: root.tagHeaderText
+            elide: Text.ElideRight
+            font.weight: Font.Medium
+        }
+        Rectangle {
```

Comment:
Could it be possible to have it inside `background` instead?

Reply:
It's a bit more than just an ExpandableTag. it also has the title in it and this is why we can't have it in the background.